src/main/scala/io/iohk/ethereum/vm/ProgramError.scala

```diff
@@ -9,6 +9,7 @@ case class  InvalidOpCode(code: Byte) extends ProgramError {
     f"${getClass.getSimpleName}(0x${code.toInt & 0xff}%02x)"
 }
 case object OutOfGas extends ProgramError
+case object InvalidJump extends ProgramError
```

Comment:
It may be convenient for experimenting and debugging to add jump target as a parameter. Take a look at `InvalidOpCode`. Valid code offset will be at most 4 hex digits long, so print it zero-padded (`f"0x$dest%04x"`).